### PR TITLE
Web Page Clone of Newsweek we page 

### PR DESCRIPTION
**Pull request by abdelp**
Responsive design of Newsweek webpage

Based on the Odin Project Requirements specified here: https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap

use of grids with bootstrap framework v4.4.1

### CODE REVIEW

**Comment by pelzolga123** [comment]
### Hi Abdel! I am marking this review request as invalid, because your linter is not setup for this repository. 
Please kindly setup linter and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by abdelp** [comment]
Enabled linter in the repository

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: assets/stylesheets/style.css [hunk 1]
```diff
@@ -0,0 +1,580 @@
+body {
+  background-color: #f4f5f6;
+}
+
+.page-header,
+.main,
+.footer {
```
Comment (addod19) [inline assets/stylesheets/style.css:11]: I suggest you give these classes a ```width:100%```. To solve the horizontal scroll we get on tablet and mobile screen sizes

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴 

Hi @abdelp, you did a great job enabling linter for your repo. 👍 However, there are some few recommendations. Find comment below:
- Kindly fix the warning
sl
 returned by [HTML markup](https://validator.w3.org/)
- ```Tablet mode``` Currently this is how it looks **(1 column)**
new1 It should be like this **(2 columns)**
onew1
- Currently,  this part looks like this
not
It should be this
an
- Kindly make the following changes and aligns on breakpoints:
Featured Stories, Top story, Opinion, Latest news, more stories, In the magazine, editors pick, Featured slidshows
co
- There is a horizontal scroll. 

Kindly make these changes and submit another code review 🙂 
Happy coding 💻 
Daniel Larbi Addo

**Comment by abdelp** [comment]
Removed warning from the linter
Updated the alignment for medium sizes devices on top history
fixed the horizontal scroll
Updated alignment for sections

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi @abdelp !
### Nice job! Project looks! :+1: 

### But your project needs a few changes before it can be approved.
#### Implementation details


*  It adds [viewport metatag](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag) 
screenshot from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap)
*  It uses Bootstrap Css classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)) 
screenshot from [Pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/)

### Main content

**It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))

* Kindly recommended checking all parts of your project, it still has a big difference:

#### Your version
us1
#### Original
us

#### Your version
subscr

#### Original
subs
newsl


*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project still needs changes

**Review by nikasulo — APPROVED** [review]
# Your hard work has paid off! :rocket: 
This is really great work @abdelp :handshake: Please merge this and be on your way :smile:. See you at the top :medal_military:  

#### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by abdelp** [comment]
Thank you very much, guys!

**Pull request merged**